src/internal/observable/dom/fetch.ts

```diff
@@ -1,4 +1,18 @@
 import { Observable } from '../../Observable';
+import { Subscription } from '../../Subscription';
+import { from } from '../../observable/from';
+import { ObservableInput } from '../../types';
+
+export function fromFetch(
+  input: string | Request,
+  init?: RequestInit
+): Observable<Response>;
+
+export function fromFetch<T>(
+  input: string | Request,
+  init: RequestInit | undefined,
+  selector: (response: Response) => ObservableInput<T>
```

Comment:
Looking at most of our uses, an overload for `fromFetch(input, selector)` without `init` would be useful (but not necessary)

Reply:
Done